contracts/contracts/strategies/NativeStaking/CompoundingValidatorManager.sol

```diff
@@ -840,6 +882,114 @@ abstract contract CompoundingValidatorManager is Governable {
             }
         }
 
+        uint256 depositsCount = depositsRoots.length;
+        uint256 totalDepositsWei = 0;
+
+        // If there are no deposits then we can skip the deposit verification
+        // This section is after the validator balance verifications so an exited validator will be marked
+        // as EXITED before the deposits are verified. If there was a deposit to an exited validator
+        // then the deposit can only be removed once the validator is fully exited.
```

Comment:
nit comment addition: 
... validator is fully exited. The ETH deposited to an exiting validator remains in the `beaconState.pendingDeposits` as part of the beacon chain's `deposits_to_postpone` array inside which the deposit remains as long as the validator has not fully exited from the beacon chain. Once the validator has fully exited the postponed deposit is credited back to the strategy contract.